### PR TITLE
[tiny][framework] Put a max cap on the commission rate of the validator

### PR DESCRIPTION
**Pull request by dmitri-perelman**
Currently limiting it to 100% during validator creation or setting the next rate.

## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-framework/sources/governance/validator.move [hunk 1]
```diff
@@ -53,8 +53,13 @@ module sui::validator {
     /// Invalidworker_address field in ValidatorMetadata
     const EMetadataInvalidWorkerAddr: u64 = 7;
 
+    /// Commission rate set by the validator is higher than the threshold
+    const ECommissionRateTooHigh: u64 = 8;
+
     const EInvalidProofOfPossession: u64 = 0;
 
+    const MaxCommissionRate: u64 = 1_000_000; // Max rate is 100%, which is 1M base points
```
Comment (emmazzz) [inline crates/sui-framework/sources/governance/validator.move:61]: I'm pretty sure 1% is 100 basis points so 100% should be 10000 basis points.

Comment (dmitri-perelman) [reply crates/sui-framework/sources/governance/validator.move:61]: Ouch, right. Max commission rate of 10000% is too much

File: crates/sui-framework/sources/governance/validator.move [hunk 2]
```diff
@@ -53,8 +53,13 @@ module sui::validator {
     /// Invalidworker_address field in ValidatorMetadata
     const EMetadataInvalidWorkerAddr: u64 = 7;
 
+    /// Commission rate set by the validator is higher than the threshold
+    const ECommissionRateTooHigh: u64 = 8;
+
     const EInvalidProofOfPossession: u64 = 0;
 
+    const MaxCommissionRate: u64 = 10_000; // Max rate is 100%, which is 10K base points
```
Comment (emmazzz) [inline crates/sui-framework/sources/governance/validator.move:61]: For this type of constant we normally use all caps in other places in the framework, such as here: https://github.com/MystenLabs/sui/blob/a91702ba32795ae247343987bf832d866842e4fd/crates/sui-framework/sources/governance/voting_power.move#L29
Change the naming here to be consistent?

**Review by emmazzz — APPROVED** [review]
One comment re naming, otherwise good to go.

**Pull request merged**